openzwave-sys/openzwave-c/value_classes/value_id.h

```diff
@@ -8,29 +8,38 @@ typedef OpenZWave::ValueID ValueID;
 typedef ValueID::ValueGenre ValueGenre;
 typedef ValueID::ValueType ValueType;
 
+typedef struct cValueID {
```

Comment:
nit: shouldn't this be called rustValueID? Because this is really the type we use in Rust ?